_rules/SC1-2-video-media-alternative.md

```diff
@@ -0,0 +1,130 @@
+---
```

Comment:
Should atomic rules reference the guidelines if they don't map to SC? And should we then not call it SC?